### PR TITLE
#229 change submission type to  on canvas

### LINKED ISSUE #229: Canvas: Change "Submission Type" on Canvas
Currently, when assignments are created in Canvas via the MITx-Canvas integration, the Submission Type setting in Canvas is set to Online -> File Upload. This has been causing some confusion with students, and if the default could be set to "No submission", that would be preferable. 

Screen Shot 2020-09-10 at 12 44 05 PM

### PR DESCRIPTION
**Pull request by umarmughal824**
Canvas: Change "Submission Type" to `No Submission`on Canvas

### CODE REVIEW

File: lms/djangoapps/canvas_integration/client.py [hunk 1]
```diff
@@ -181,7 +181,7 @@ def create_assignment_payload(subsection_block):
                 # string representation.
                 else subsection_block.fields["due"].astimezone(pytz.UTC).isoformat()
             ),
-            "submission_types": ["online_upload"],
+            "submission_types": ["none"],
```
Comment (pdpinch) [inline lms/djangoapps/canvas_integration/client.py:184]: How did you determine this value, "none" ?

**Comment by pdpinch** [comment]
I got a follow up request to also set "Publish" to false, and it seems like we can do that in this PR as well:

https://github.com/mitodl/edx-platform/blob/a12173a71a33d382829a7038395613f4a1b6ff8e/lms/djangoapps/canvas_integration/client.py#L185

File: lms/djangoapps/canvas_integration/client.py (hunk 1 above)
Comment (pdpinch) [reply lms/djangoapps/canvas_integration/client.py:184]: If these are the relevant docs, it seems like we are limited in our options here: 

https://canvas.instructure.com/doc/api/submissions.html#method.submissions.create

Comment (umarmughal824) [reply lms/djangoapps/canvas_integration/client.py:184]: @pdpinch I got it from here
```
const override = mockOverride({
    submissionTypes: [
      'none',
      'external_tool',
      'online_upload',
      'on_paper',
      'online_text_entry',
      'online_url'
    ]
  })
```
https://github.com/instructure/canvas-lms/blob/45961e84b828b8f56e487de29e405c6ca97a1631/app/jsx/assignments_2/teacher/components/Overrides/__tests__/OverrideSubmissionTypes.test.js#L24

Comment (umarmughal824) [reply lms/djangoapps/canvas_integration/client.py:184]: I had requested to grant me access to the course page on the ticket couple and one day ago, to verify that value but did not get any response from your side.

Comment (umarmughal824) [reply lms/djangoapps/canvas_integration/client.py:184]: @pdpinch that value could be verified only from the page source that what's I need access and requesting that on the ticket for last few days.

**Comment by umarmughal824** [comment]
> I got a follow up request to also set "Publish" to false, and it seems like we can do that in this PR as well:
> 
> https://github.com/mitodl/edx-platform/blob/a12173a71a33d382829a7038395613f4a1b6ff8e/lms/djangoapps/canvas_integration/client.py#L185

sure

File: lms/djangoapps/canvas_integration/client.py (hunk 1 above)
Comment (umarmughal824) [reply lms/djangoapps/canvas_integration/client.py:184]: Screenshot 2020-12-10 at 19 21 06

**Comment by pdpinch** [comment]
@mitodl/devops how can we deploy this to mitx-qa-next for testing? Should we merge it first?

**Comment by shaidar** [comment]
@pdpinch Yeah but it needs to be against `mitx/koa` and not `mitx/juniper`. I can change it manually on qa-next instances to test it out if it hasn't been tested out elsewhere

**Comment by umarmughal824** [comment]
@pdpinch @shaidar this PR was created three days ago and at that time, canvas integration was not a part of the `mitx/koa` that was got merged just two days ago. So I created that PR for `mitx/juniper`.

Let me know what's the plan now. In my opinion, we should merge it and open up a cherry-pick PR in `mitx/koa`.

**Comment by shaidar** [comment]
@umarmughal824 I don't see the point of merging into `mitx/juniper` since we won't be deploying that branch. I would rebase on `mitx/koa` and then we can merge and deploy to QA unless @blarghmatey has other ideas.

**Comment by blarghmatey** [comment]
I agree that it makes sense to close this PR and then apply the diff to the mitx/koa branch instead.

**Pull request closed without merging**